src/pe_reports/report_gen/views.py

```diff
@@ -0,0 +1,191 @@
+"""Classes and associated functions that render the UI app pages."""
+
```

Comment:
Use LOGGER instead of print statements.

Reply:
addressed in attatched commit: [6dfad1b](https://github.com/cisagov/pe-reports/pull/282/commits/6dfad1b08195be4a60b71de96b227e0488931edc)